src/components/v5/shared/SubNavigationItem/types.ts

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { MessageDescriptor } from 'react-intl';
+import { IconSize } from '~shared/Icon/Icon';
```

Comment:
Can we group these imports correctly please

Reply:
fixed :)